mmtrack/datasets/vot_dataset.py

```diff
@@ -1,8 +1,19 @@
+import os.path as osp
+
 import numpy as np
+from mmcv.utils import print_log
 from mmdet.datasets import DATASETS
 
+from mmtrack.core.evaluation import eval_sot_accuracy_robustness, eval_sot_eao
 from .sot_test_dataset import SOTTestDataset
 
+# parameter, used for EAO evaluation, may vary by different vot challenges.
+INTERVAL = dict(
+    vot2018=[100, 356],
+    vot2019=[46, 291],
+    vot2020=[115, 755],
+    vot2021=[115, 755])
```

Comment:
move `INTERVAL`  into `VOTDataset`, and set it as a property. (self.INTERVAL=...)